### DISCUSSION TITLE
Filter by string length

### DISCUSSION THREAD

**Discussion by maitrungduc1410** [prisma/prisma#16277] [Q&A] [answered]
Hi all,

Can we filter a table by a column's length without using `queryRaw`?

Something like this:
```js
prisma.users.findMany({
  where: {
    // "fullname" >= 20 
  }
})
```

**Comment by nurul3101** [comment, marked as answer]
Hey @maitrungduc1410 👋 

For now you would need to use Raw Query. Here's a detailed StackOverflow [Answer](https://stackoverflow.com/questions/71718442/is-it-possible-in-prisma-to-filter-by-string-length) which describes the process of how you can filter by string length.
We do have support for _less than_, _greater than_ and other similar comparison operators as described in the docs [here] (https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#lt), but I don't think they will be helpful in this particular case.

**Reply by maitrungduc1410** [reply]
Got it, thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
